v1/linkml-schemas/common.yaml

```diff
@@ -74,12 +74,15 @@ slots:
     range: date
     slot_uri: schema:validUntil
   iss:
+    required: true
     range: uri
     description: The `iss` (issuer) claim identifies the principal that issued the JWT.
   sub:
+    required: true
     range: uri
     description: The `sub` (subject) claim identifies the principal that is the subject of the JWT.
   nbf:
+    required: true
```

Comment:
In general whether a particular JWT claim is required or not is a very much context-dependent. So I think this should be in slot_usage or similar.

This should follow RFC 7519 which specifically says
> None of the claims defined below are intended to be mandatory to use or implement in all cases, but rather they provide a starting point for a set of useful, interoperable claims.